docs/excel/excel-add-ins-pivottables.md

```diff
@@ -175,6 +176,67 @@ The following code samples changes the aggregation to be averages of the data.
     });
 ```
 
+## Change calculations with a ShowAsRule
+
+PivotTables, by default, aggregate the data of their row and column hierarchies independently. A `ShowAsRule` changes the data hierarchy to output values based on other items in the PivotTable.
+
+The `ShowAsRule` object has three properties:
+-	`calculation`: The type of relative calculation to apply to the data hierarchy (the default is `none`).
+-	`baseField`: The field within the hierarchy containing the base data before the calculation is applied. The `PivotField` usually has the same name as its parent hierarchy.
+-	`baseItem`: The individual item compared against the values of the base fields based on the calculation type. Not all calculations require this field.
+
+The following example sets the calculation on the **Sum of Crates Sold at Farm** data hierarchy to be a percentage of the column total. 
```

Comment:
Feels disjointed to me that this first example is structured as: 1) image of result, 2) code that caused that result -- while the next example that follows is structured in reverse: 1) code that causes the result, 2) image of result.  Suggest that you structure these examples the same with respect to relative sequence of image and code example.

Reply:
@AlexJerabek restructuring looks good. I'm going to approve this PR, but as I stated earlier -- I'm not very familiar with the subject matter, so would suggest that you wait for Juan's feedback before merging.